src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -285,6 +297,30 @@ protected void interrupted() {
         };
     }
 
+    public Command UpSpecified(double inches) {
```

Comment:
Rename to LiftByInches or something similar

Reply:
done